### PR TITLE
Added Michael Dawson's name (for consistency)

### PR DESCRIPTION
**Pull request by ThePrez**
For consistent format with the others listed.

### CODE REVIEW

File: locale/en/get-involved/node-speakers.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ If anyone is reported as malicious or making others feel uncomfortable, they may
 #### British Columbia
 
 #### Ontario 
+
+##### Michael Dawson
```
Comment (richardlau) [inline locale/en/get-involved/node-speakers.md]:
```suggestion
###### Michael Dawson
```

File: locale/en/get-involved/node-speakers.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ If anyone is reported as malicious or making others feel uncomfortable, they may
 #### British Columbia
 
 #### Ontario
```
Comment (richardlau) [inline locale/en/get-involved/node-speakers.md:22]:
```suggestion
 ##### Ontario 
```

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by ZYSzys** [comment]
Ping @ThePrez , could you please land @richardlau's suggestions ?

**Review by willin — APPROVED** [review]

**Comment by ThePrez** [comment]
Thanks, @richardlau, for noting my header level incorrectness. I'm not good at details and I think I was thrown off by the misplaced "British Columbia" header (also possibly the lack of a city for Ontario), both of which I've corrected in my latest commits.

**Review by richardlau — APPROVED** [review]

**Review by ZYSzys — APPROVED** [review]

**Comment by ZYSzys** [comment]
Thanks👍

**Pull request merged**